config_templates/gretel/synthetics/navigator-ft-differential-privacy.yml

```diff
@@ -0,0 +1,40 @@
+schema_version: 1.0
+name: "navigator_ft-dp"
+models:
+- navigator_ft:
+    data_source: __tmp__
+
+    # Optionally group records by the column(s) set below.
+    # This is useful if you need to maintain correlations
+    # across multiple records. Otherwise, the training
+    # assumes records are independent.
+    group_training_examples_by: null
+
+    # Optionally order records by the column set below.
+    # This is useful if your records are sequential.
+    # Note that this parameter can only be used when
+    # your records are grouped using the above parameter.
+    order_training_examples_by: null
+
+    generate:
+        num_records: 1000
```

Comment:
We are reducing num_records to 1000 to keep generation time lower, right? Just note this will have implications for the SQS report - we'll end up using a sample of 1000 instead of the usual 5000

Reply:
Changed it to 5000, but I'll leave 1000 in the notebooks if that's okay